src/backend/blas.c

```
@@ -271,6 +271,28 @@ void cml_blas_free(CMLBlasContext* ctx) {
     free(ctx);
 }
 
+void cml_blas_set_num_threads(int n) {
```

Cppcheck (2.21.0)

[style] The function 'cml_blas_set_num_threads' is never used.

(unusedFunction)